content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,225 @@ section: doc
++
+image::tutorials/IBMCloud/details-resource.png[Resource details]
+
+. Give *tags* to your Jenkins, for more information visit link:https://cloud.ibm.com/docs/account?topic=account-tag[tags].
++
+image::tutorials/IBMCloud/details-tags.png[Default value parameters]
+
+. Select *Parameters with default values*.
+You can set custom deployment values or use the default ones.
++
+image::tutorials/IBMCloud/parameters.png[IBM parameters setup]
+
+. Set the Jenkins password in the parameters.
++
+image::tutorials/IBMCloud/password.png[create password instruction]
+
+. After finishing everything, *select* the box next to the agreements and select *install*.
++
+image::tutorials/IBMCloud/install.png[Installation instructions]
+
+. The Jenkins workspace will start installing, wait a couple of minutes for this to complete.
++
+image::tutorials/IBMCloud/in-progress.png[workspace installation screen]
+
+. Your  Jenkins workspace has been successfully deployed.
++
+image::tutorials/IBMCloud/done.png[IBM workspace deployment screen]
+
+== Verify Jenkins installation
+
+. Go to link:http://cloud.ibm.com/resources[Resources] in your browser.
+. Select *Clusters*.
+. Select your cluster.
++
+image::tutorials/IBMCloud/resource-select.png[Resource selection screen]
+
+. Now you are at you clusters overview.
+... Select *Actions* in the top right corner of the page.
+... Select *Web terminal* in the dropdown menu.
++
+image::tutorials/IBMCloud/cluster-main.png[Clusters overview page]
+
+. Select *Install* and then wait for a couple of minutes.
++
+image::tutorials/IBMCloud/terminal-install.jpg[terminal install screen]
+
+. Once installation has completed, you will repeat this process.
+
+... Select *Actions*.
+... Select *Web terminal* and a terminal window will open.
+
+. *Type* in the terminal, be sure to change NAMESPACE to the namespace you choose at the deployment setup:
++
 [source,bash]
 ....
 $ kubectl get ns
 ....
-
-image::tutorials/IBMCloud/get-ns.png[]
-
++
+image::tutorials/IBMCloud/get-ns.png[get-ns command]
++
 [source,bash]
 ....
 $ kubectl get pod -n NAMESPACE -o wide
 ....
-
-image::tutorials/IBMCloud/get-pods.png[]
-
++
+image::tutorials/IBMCloud/get-pods.png[get-pods command]
++
 [source,bash]
 ....
 $ kubectl get service -n NAMESPACE
 ....
-
-image::tutorials/IBMCloud/get-service.png[]
-
-* Running Jenkins service will be visible
-* Copy the **External ip**, you can access the website on this IP
-* Paste it into your browser
-* Jenkins login portal will be visible
-
-image::tutorials/IBMCloud/login.png[]
-
-* Please enter your Username ( default is user) and your password which you set at the deployment phase
-
-image::tutorials/IBMCloud/welcome.png[]
++
+image::tutorials/IBMCloud/get-service.png[get-service command]
+
+ * Running Jenkins service will be visible.
+. Copy the *External-IP*, you can access the website on this IP.
+. Paste it into your browser.
+. The Jenkins login portal will be visible.
```

Comment:
We could adjust this slightly to remove the list format from the last step, since it's the expected result of completing these steps, and change it to present tense.

```suggestion
. Copy the **External-IP**. You can access the website on this IP.
. Paste it into your browser.
+
The Jenkins login portal should now be visible.
```